extensions/amp-story/amp-story.md

```diff
@@ -1386,6 +1386,177 @@ The `<amp-story-bookend>` must have a `src` attribute pointing to the JSON confi
+        </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="bacon-answer">
+        <amp-story-grid-layer>
+          <p> Bacon, of course! </p>
+        </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="boba-answer">
+        <amp-story-grid-layer>
+          <p> Boba, because it's a perfect drink! </p>
+        </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="pizza-answer">
+        <amp-story-grid-layer>
+        <p> Pizza, it's a true classic! </p>
+        </amp-story-grid-layer>
+    </amp-story-page>
+    ...
+</amp-story>
+```
+
+### The `advance-to` Attribute
+The `advance-to` attribute allows you to specify what page should follow a given `amp-story-page` when the user advances through the story. Recall that without branching the next page is by default the next `amp-story-page` in the DOM. This is still the default behavior should you choose not to use this attribute when creating non-linear stories.
+
+<table>
+<thead>
+<tr>
+<th>Attribute</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>advance-to</td>
+<td>Specifies what page should follow when the user advances next. The id of the page that should follow is required.</td>
+</tr>
+</tbody>
+</table>
+
+The following is a modification of the previous [story with a quiz experience](#quiz-example). By using the `advance-to` attribute, we create a fallback page to navigate to should the user not select any of the food items listed.
+
+```html
+<amp-story id="story" standalone>
+    <amp-story-page id="cover" advance-to="fallback-answer">
+        <amp-story-grid-layer>
+          <p> What is your favourite food? </p>
+          <button on="tap:story.goToPage(id=bacon-answer)"> Bacon! </button>
+          <button on="tap:story.goToPage(id=boba-answer)"> Boba! </button>
+          <button on="tap:story.goToPage(id=pizza-answer)"> Pizza! </button>
+        </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="bacon-answer">
+        <amp-story-grid-layer>
+          <p> Bacon, of course! </p>
+        </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="boba-answer">
+        <amp-story-grid-layer>
+        <p> Boba, because it's a perfect drink! </p>
+        </amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="pizza-answer">
+        <amp-story-grid-layer>
+          <p> Pizza, it's a true classic! </p>
+        </th>amp-story-grid-layer>
+    </amp-story-page>
+
+    <amp-story-page id="fallback-answer">
+        <amp-story-grid-layer>
+          <p> It looks like you like something else. You're very unique! </p>
+        </amp-story-grid-layer>
+    </amp-story-page>
+    ...
+</amp-story>
+```
+If we had not used the `advance-to` attribute with the specified value and the user chose not to tap on any of the buttons on the first page of the story and advance forward, they would see the `bacon-answer` page.
```

Comment:
Update "If we had not used the..." 
to 
"If the `advance-to` attribute had not been used with the `"fallback-answer"` value and the user advanced forward without selecting a choice, the story would have presented the the `bacon-answer` page."

Reply:
Done.